.terraform.lock.hcl

```diff
@@ -0,0 +1,21 @@
+# This file is maintained automatically by "terraform init".
+# Manual edits may be lost in future updates.
+
```

Comment:
Can you put a note on https://projects.ucd.gpn.gov.uk/browse/DW-8217 that this .terraform.lock.hcl has already been committed as part of this IPAM work please.
Just makes it easier for when people come to pick up the ticket.